CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.25.0-rc.0 / 2021-02-12
+
+* [FEATURE] API/Experimental: Accept remote_write requests. #8424
+* [FEATURE] PromQL/Experimental: Add '@ <timestamp>' selector. #8121 #8436 #8425
+* [FEATURE] Experimental: Add exemplars support. #6635
+* [ENHANCEMENT] Add optional name property to testgroup for better test failure output. #8440
+* [ENHANCEMENT] Add warnings into React Panel on the Graph page. #8427
+* [ENHANCEMENT] TSDB: Increase the number of buckets for the  compaction duration metric. #8342
+* [ENHANCEMENT] Remote: Allow passing along custom remote_write HTTP headers. #8416
+* [ENHANCEMENT] Mixins: Scope grafana configuration. #8332
+* [ENHANCEMENT] Kubernetes SD: Add endpoint labels metadata. #8273
+* [ENHANCEMENT] UI: Expose total number of label pairs in head in TSDB stats page. #8343
+* [ENHANCEMENT] TSDB: Reload blocks every minute, to detect new blocks and enforce retention more often. #8343
+* [BUGFIX] API: Fix global URL when external address has no port. #8359
+* [BUGFIX] Backfill: Fix error message handling. #8432
+* [BUGFIX] Deprecate unused flag --alertmanager.timeout. #8407
+* [BUGFIX] Remote: Adapt metric name in remote-write alert rules mixin. #8423
```

Comment:
This should probably be labeled as `Mixins` as there were no changes to the Remote code for it. Also, it is pretty vague, perhaps `Mixins: Support remote-write metrics renamed in v2.23 in alerts`?